handlers.go

```diff
@@ -4,44 +4,102 @@ import (
 	"fmt"
 	"net/http"
 
+	"encoding/json"
 	"github.com/Financial-Times/go-fthealth/v1a"
-	"regexp"
+	log "github.com/Sirupsen/logrus"
```

Comment:
Can we use the default `log` instead of `logrus`? As I understood correctly, logrus isn't very performance-friendly